pkg/apis/apps/v1beta2/conversion.go

```diff
@@ -173,48 +130,31 @@ func Convert_apps_Deployment_To_v1beta2_Deployment(in *apps.Deployment, out *app
 		delete(out.Annotations, appsv1beta2.DeprecatedRollbackTo)
 	}
 
-	if err := Convert_apps_DeploymentStatus_To_v1beta2_DeploymentStatus(&in.Status, &out.Status, s); err != nil {
-		return err
-	}
 	return nil
 }
 
 func Convert_apps_DaemonSet_To_v1beta2_DaemonSet(in *apps.DaemonSet, out *appsv1beta2.DaemonSet, s conversion.Scope) error {
-	out.ObjectMeta = in.ObjectMeta
-	out.Annotations = deepCopyStringMap(out.Annotations)
-	out.Annotations[appsv1beta2.DeprecatedTemplateGeneration] = strconv.FormatInt(in.Spec.TemplateGeneration, 10)
-	if err := Convert_apps_DaemonSetSpec_To_v1beta2_DaemonSetSpec(&in.Spec, &out.Spec, s); err != nil {
-		return err
-	}
-	if err := s.Convert(&in.Status, &out.Status, 0); err != nil {
+	if err := autoConvert_apps_DaemonSet_To_v1beta2_DaemonSet(in, out, s); err != nil {
 		return err
 	}
+
+	out.Annotations = deepCopyStringMap(out.Annotations)
+	out.Annotations[appsv1beta2.DeprecatedTemplateGeneration] = strconv.FormatInt(in.Spec.TemplateGeneration, 10)
 	return nil
 }
 
 func Convert_apps_DaemonSetSpec_To_v1beta2_DaemonSetSpec(in *apps.DaemonSetSpec, out *appsv1beta2.DaemonSetSpec, s conversion.Scope) error {
```

Comment:
delete if just delegating to the autogenerated method

Reply:
I did that initially, but unfortunately I can't - code-generator doesn't generate the public method for it, because auto-generated ones contains a warning.

Reply:
done